service/updater/updater.go

```diff
@@ -16,48 +16,48 @@ import (
 // Config represents the configuration used to create a new updater.
 type Config struct {
 	// Dependencies.
-	KubernetesClient *kubernetes.Clientset
-	Logger           micrologger.Logger
+	K8sClient kubernetes.Interface
```

Comment:
The oepratorkit way of creating the k8s client made the interface change necessary. Then I also changed the naming a bit to also align with other projects.